pkg/kubelet/kubelet_pods.go

```diff
@@ -232,6 +233,12 @@ func hostsFileContent(hostIP, hostName, hostDomainName string) []byte {
 	} else {
 		buffer.WriteString(fmt.Sprintf("%s\t%s\n", hostIP, hostName))
 	}
+	// write each IP/hostname pair as an entry into hosts file
+	for _, hostAlias := range hostAliases {
+		for _, hostname := range hostAlias.Hostnames {
+			buffer.WriteString(fmt.Sprintf("%s\t%s\n", hostAlias.IP, hostname))
```

Comment:
Comment that this was previously validated to be a valid IP address and hostname?